From 922b6040eb447db5db95bee8e8283829bfa64e7d Mon Sep 17 00:00:00 2001
From: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>
Date: Fri, 12 May 2023 09:22:36 -0700
Subject: [PATCH 2/2] meson/vaon12: fix driver file name for mingw build

This fixes vaon12 driver file name to be consistent with libva
expectation - vaon12_drv_video.dll - without lib prefix.

Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/22995>
---
 src/gallium/targets/va/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/targets/va/meson.build b/src/gallium/targets/va/meson.build
index b4cfbfb70e8..c6c50bccb19 100644
--- a/src/gallium/targets/va/meson.build
+++ b/src/gallium/targets/va/meson.build
@@ -73,6 +73,7 @@ if host_machine.system() == 'windows'
     link_depends : va_link_depends,
     install : true,
     name_suffix : 'dll',
+    name_prefix : '',  # otherwise mingw will create libvaon12_drv_video.dll
   )
 else
   libva_gallium = shared_library(
-- 
2.41.0

